doc/get_started/linux-eve-raspberry-c.md

```diff
@@ -0,0 +1,137 @@
+
+        sudo apt-get update
+
+        sudo apt-get install -y curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake git unzip openjdk-7-jre
+    
+    ***Note:*** *This setup process requires cmake version 3.0 or higher.* 
+    
+    *You can verify the current version installed in your environment using the  following command:*
+
+        cmake --version
+
+    *For information about how to upgrade your version of cmake to 3.2 on Ubuntu 14.04, see <http://askubuntu.com/questions/610291/how-to-install-cmake-3-2-on-ubuntu-14-04>.*
+    
+    *This library also requires gcc version 4.9 or higher. You can verify the current version installed in your environment using the following command:*
+    
+        gcc --version 
+
+    *For information about how to upgrade your version of gcc on Ubuntu 14.04, see <http://askubuntu.com/questions/466651/how-do-i-use-the-latest-gcc-4-9-on-ubuntu-14-04>.*
+
+-  Download the Microsoft Azure IoT Device SDK for C to the board by issuing the following command on the board::
+
+        git clone --recursive https://github.com/Azure/azure-iot-sdks.git
+
+-  Verify that you now have a copy of the source code under the directory ~/azure-iot-sdks.
+
+-   Edit the following file using any text editor of your choice:
+  
+    **For AMQP protocol:**
+
+        azure-iot-sdks/c/iothub_client/samples/iothub_client_sample_amqp/iothub_client_sample_amqp.c
+
+    **For HTTP protocol:**
+
+        azure-iot-sdks/c/iothub_client/samples/iothub_client_sample_http/iothub_client_sample_http.c
+	
+    **For MQTT protocol:**
+		
+        azure-iot-sdks/c/iothub_client/samples/iothub_client_sample_mqtt/iothub_client_sample_mqtt.c
+
+-   Find the following place holder for IoT connection string:
+
+        static const char* connectionString = "[device connection string]";
+
+-   Replace the above placeholder with device connection string you obtained in [Step 1](#Step-1:-Prerequisites) and save the changes.
+
+-   Build the SDK using following command.
+	
+        sudo ./azure-iot-sdks/c/build_all/linux/build.sh | tee LogFile.txt
+
+## 3.2 Send Device Events to IoT Hub:
+
+-   Run the sample by issuing following command:
+
+    **For AMQP protocol:** Run sample iothub_client_sample_amqp
+
+        ~/cmake/iothub_client/samples/iothub_client_sample_amqp/iothub_client_sample_amqp
+
+    **For HTTP protocol:** Run sample iothub_client_sample_http
+
+        ~/cmake/iothub_client/samples/iothub_client_sample_http/iothub_client_sample_http
+
+ 	  **For MQTT protocol:** Run sample iothub_client_sample_mqtt
+
+        ~/cmake/iothub_client/samples/iothub_client_sample_mqtt/iothub_client_sample_mqtt
+
+-   On Windows, refer "Send cloud-to-device messages" in [DeviceExplorer Usage document][lnk-device-explorer] for instructions on sending messages to device.
+
+-   If you are running other OS, please use the JavaScript tool [iot-hub explorer tool][lnk-iothub-explorer]
```

Comment:
Typically the iothub explorer is the first one mentioned.
I believe Dominic has made a bunch of changes to make this more generic. If you look at the other docs you probably can align with that.